### DISCUSSION TITLE
Name of a system

### DISCUSSION THREAD

**Discussion by Graumenth** [flybywiresim/aircraft#8688] [General]
Hello,

What is the name of the system which recommends flaps on ND in the approach phase? I tried to find it in FCOM but I couldn't.

**Comment by alepouna** [comment]
Are you refering to the A320 or A380? I am not sure such display/recommendation exists for the ND (Navigation Display) regardless of aircraft

**Comment by tracernz** [comment]
This is part of the CDFA feature on the Honeywell FMS (H3 or newer).

**Reply by Graumenth** [reply]
Thank you. Yes. It's called "CDA function" in FCOM.
